homeworks/rostyslav.havuka_RostyslavHavuka/12-httpclient/user-card/src/app/app.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { finalize } from 'rxjs/operators';
+import { User } from './models/user.model';
+import { UserService } from './services/user.service'
+import { LoaderService } from './services/loader.service';
+
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent implements OnInit{
+  users: User[] = [];
+  items: Array<number> = [1, 2, 3];
+  isLoading: Subject<boolean> = this.loaderService.isLoading;
+  isContent!: boolean;
+  constructor(private userService: UserService, private loaderService: LoaderService) {}
+
+  ngOnInit() {
+    this.getUsers(0,2)
+    this.isLoading.subscribe((e) => console.log(e));
+  }
+
+  getUsers(a: number, b: number) {
```

Comment:
What is `a` and `b`? :D
It doesn't make sence! Rename the variables to be more readable and understandable